eland/dataframe.py

```diff
@@ -793,12 +866,15 @@ def to_string(self, buf=None, columns=None, col_space=None, header=True,
             else:
                 max_rows = min(num_rows, max_rows)
         elif max_rows is None:
-            warnings.warn("DataFrame.to_string called without max_rows set "
-                          "- this will return entire index results. "
-                          "Setting max_rows={default}"
-                          " overwrite if different behaviour is required."
-                          .format(default=DEFAULT_NUM_ROWS_DISPLAYED),
-                          UserWarning)
+            warnings.warn(
```

Comment:
to f-string - I'm not going to review more for this, as finding '.format(' and changing would be more effective